app/elements/markdown/render-markdown.html

```diff
@@ -19,8 +21,15 @@
 					observer: '_updateChanges'
 				}
 			},
+                        created: function() {
+                                marked.setOptions({
+                                        highlight: function (code) {
+                                            return hljs.highlightAuto(code).value;
```

Comment:
Using 4 spaces instead of 8 above and below. Also, we use tabs, please adjust your indentation characters.